util/pastPerformance/testReleasesPerformance

```diff
@@ -34,10 +34,10 @@
 #
 
 function testReleasePerformance {
-    pushd $CHPL_HOME
+    pushd $CHPL_HOME > /dev/null
```

Comment:
What's the goal of piping this output to /dev/null?  start_test output is already going to be huge, so there's no strong motivation to be non-verbose that I can see...  (and maybe the output would be useful to search on when looking for where each phase of testing occurs?)